chipy_org/settings.py

```diff
@@ -61,6 +62,12 @@ def env_list(key, defaults=None, delimiter=","):
 
 MANAGERS = ADMINS
 
+
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
I probably have to try this out.  We do have media url and media root defined further down in the file.  It could be fine but these at least have to be in the same place in the file.